costmap_2d/plugins/obstacle_layer.cpp

```diff
@@ -130,15 +130,21 @@ void ObstacleLayer::onInitialize()
       source_node.getParam(raytrace_range_param_name, raytrace_range);
```

Comment:
We should update this to search for max_raytrace_range, if not found, it looks for raytrace_range, but prints a warning that raytrace_range is deprecated (and yes, this starts to leak into the config for each robot, but we will give several/many release cycles before we remove the deprecated part)